topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.md

```diff
@@ -0,0 +1,514 @@
+
+![Bandage output](../../images/plasmid-metagenomics-nanopore/Bandage_output.png)
+
+# Species and plasmids
+
+## Prediction of plasmid sequences and classes using PlasFlow
+
+To automatically determine whether the contigs represent chromosomal or plasmid DNA, **PlasFlow** {% icon tool %} ({% cite Krawczyk2018 %}) can be used, also in the case where a full circular plasmid sequence was not assembled. Furthermore, it assigns the contigs to a bacterial class.
+
+**PlasFlow** {% icon tool %} is a set of scripts used for prediction of plasmid sequences in metagenomic contigs.
+It relies on the neural network models trained on full genome and plasmid sequences and is able to differentiate between plasmids and chromosomes with accuracy reaching 96%.
+
+![Pairwise alignment](../../images/plasmid-metagenomics-nanopore/PlasFlow.png)
+
+
+> ### {% icon hands_on %} Hands-on: Prediction of plasmid sequences
+>
+> 1. **PlasFlow** {% icon tool %} with the following parameters
+>   - *"Sequence Reads"*: the `Final Assembly` collection created by **Unicycler** {% icon tool %}
+>
+> > ### {% icon question %} Question
+> >
+> > What is the classification of contig_id 0 in RB10? (Hint: Check the probability table created by PlasFlow)
+> >
+> > > ### {% icon solution %} Solution
+> > > plasmid.Proteobacteria
+> > >
+> > > This can be determined by looking at the 5th column of the probability table.
+> > {: .solution }
+> {: .question}
+{: .hands_on}
+
+
+The most important output of **PlasFlow** {% icon tool %} is a tabular file containing all predictions (specified with `--output` option), consisting of several columns including:
+
+```
+contig_id 	contig_name 	contig_length 	id 	label 	...
+```
+
+where:
+
+- `contig_id`is an internal id of sequence used for the classification
+- `contig_name` is a name of contig used in the classification
+- `contig_length` shows the length of a classified sequence
+- `id` is an internal id of a produced label (classification)
+- `label` is the actual classification
+- `...` represents additional columns showing probabilities of assignment to each possible class
+
+Additionally, PlasFlow produces FASTA files containing input sequences binned to plasmids, chromosomes and unclassified.
+
+# Antibiotic Resistance
+
+## Scan genome contigs for antimicrobial resistance genes
+
+To determine whether the contigs contain antimircobial resistance genes (AMR) [staramr](https://github.com/phac-nml/staramr) can be used.
+**Staramr** {% icon tool %} (*AMR) scans bacterial genome contigs against both the **ResFinder** {% icon tool %} ({% cite Zankari2012 %}), **PointFinder** {% icon tool %} ({% cite Zankari2017 %}) and **PlasmidFinder** {% icon tool %} ({% cite Carattoli2014 %})  databases (used by the ResFinder webservice)
```

Comment:
ResFinder and Co are the databases, so the tool-icon could be confusing here.

Reply:
```suggestion
**Staramr** {% icon tool %} (*AMR) scans bacterial genome contigs against  the **ResFinder** ({% cite Zankari2012 %}), **PointFinder** ({% cite Zankari2017 %}) and **PlasmidFinder** ({% cite Carattoli2014 %})  databases (used by the ResFinder webservice)
```